modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/wifi.lua

```diff
@@ -491,6 +498,17 @@ if hwtype == "mac80211" then
 
 	ifname = s:taboption("advanced", Value, "ifname", translate("Interface name"), translate("Override default interface name"))
 	ifname.optional = true
+	
+	short_preamble = s:taboption("advanced", Flag, "short_preamble", translate("Short Preamble"))
+    short_preamble.default = short_preamble.enabled
+
+    dtim_period = s:taboption("advanced", Value, "dtim_period", translate("DTIM Interval"),"Delivery Traffic Indication Message Interval")
+    dtim_period.optional = true
+    dtim_period.placeholder = 2
+
+    disassoc_low_ack = s:taboption("advanced", Flag, "disassoc_low_ack", translate("Disassociate On Low Acknowledgement"),
```

Comment:
Please indent using tabs and add `disassoc_low_ack` to the list of local variables at the top